protocol_driver_test.cc

```diff
@@ -114,6 +114,20 @@ TEST_P(ProtocolDriverTest, Invoke) {
     });
   }
   pd->ShutdownClient();
+
+  if (pdo.protocol_name() == "double_barrel") {
```

Comment:
this should be a separate test function.

Reply:
Made a new file composable_protocol_driver_test.cc